tests/pii_replacer/test_patterns.py

```
@@ -0,0 +1,753 @@
+        persona_backend="faker",
+        sdg_pgms_src="/tmp",
+        managed_assets_path=None,
+    )
+    out, _ = run_replacement(df, plan, runtime)
+    return out["date_of_birth"]
+
+
+def test_a_column_of_one_date_format_keeps_it():
+    dates = [f"{(i % 12) + 1:02d}/{(i % 28) + 1:02d}/1980" for i in range(10)]
+    result = _dob_replacement(["%m/%d/%Y"], dates)
+    for original, new in zip(dates, result):
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)